### DISCUSSION TITLE
[Feature] Allow "Select All Photos but only Upload after today"

### DISCUSSION THREAD

**Discussion by twitsforbrains** [immich-app/immich#2012] [Feature Request]
### The feature

I used the bulk upload CLI to upload a bunch of photos I have backed up from other services including my phone. I ran a script to update exif data on all the photos just so that I could have a nice timeilne.

Now when I use my phone, every new photo I take, I need to "Choose which photos Immich can see". I don't want to use "Select all" because since I changed exif data, I suspect all my photos will be duplicated in immich. 

I was wondering if it is possible to have an option to "Give all photo access to immich but only upload everything after today". That way, I can simply "select all photos" once for Immich (and not have to worry about duplicates) instead of what I'm currently doing which is manually selecting new photos every few days. 

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Closing as a duplicate of #4169 since that's had more activity.
